src/ucs/config/ucm_opts.c

```diff
@@ -60,6 +60,11 @@ static ucs_config_field_t ucm_global_config_table[] = {
    ucs_offsetof(ucm_global_config_t, enable_syscall),
    UCS_CONFIG_TYPE_BOOL},
 
+  {"ENABLE_BISTRO_HOOKS", "no",
```

Comment:
All our vars have _ENABLE suffix in the end, like
```
$ucx_info -c | grep -i enable
UCX_LOG_PRINT_ENABLE=n
UCX_RC_VERBS_FC_ENABLE=y
UCX_RC_VERBS_TM_ENABLE=n
UCX_RC_MLX5_FC_ENABLE=y
UCX_RC_MLX5_TM_ENABLE=n
UCX_DC_VERBS_FC_ENABLE=n
UCX_DC_VERBS_TM_ENABLE=n
UCX_DC_MLX5_FC_ENABLE=y
UCX_DC_MLX5_TM_ENABLE=n
```
Also, would not having common variable for hook strategies be better?

Reply:
+ let's make it enabled by default

Reply:
option is renamed, enabled by default